AppCenter/AppCenterTests/MSChannelUnitDefaultTests.m

```diff
@@ -1034,7 +1034,7 @@ - (void)testPauseOnIngestionPaused {
                                  if (error) {
                                    XCTFail(@"Expectation Failed with error: %@", error);
                                  }
-                                 XCTAssertTrue([self.sut paused]);
+                                 XCTAssertTrue([self.sut isPaused]);
```

Comment:
Same here we should be able to use the `paused` property everywhere in this file.
```suggestion
                                 XCTAssertTrue([self.sut.paused]);
```